generators/base/templates/App/Components/Utilities/Environment/index.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { render } from "test-utilities";
+import Environment from "./index";
+
+it("renders a badge specific to the environment", () => {
+  const { getByText } = render(<Environment env="development" />);
+  expect(getByText("D")).toBeDefined();
+});
+
```

Comment:
Is it worth also adding [Edge and Staging](https://github.com/simpleweb/romulus-cli/blob/master/generators/base/templates/App/Components/Utilities/Environment/index.tsx#L6)?